core/variant/variant_call.cpp

```diff
@@ -2512,6 +2512,12 @@ static void _register_variant_builtin_methods() {
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "DOWN", Vector3(0, -1, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "FORWARD", Vector3(0, 0, -1));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "BACK", Vector3(0, 0, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_LEFT", Vector3(1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_RIGHT", Vector3(-1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_UP", Vector3(0, 1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_DOWN", Vector3(0, -1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_FORWARD", Vector3(0, 0, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "GLTF_BACK", Vector3(0, 0, -1));
```

Comment:
I am in favor of the `looking_at` changes, but not these. Perhaps this should be a separate PR? I think if we had a PR just for the looking_at changes this would be easier to approve.

Reply:
I suppose this is necessary as a justification for the `is_z_forward` option. The documentation also describes glTF forward (+Z). 

The +Z concept remains when it comes to importing/exporting assets. For example, it feels strange to specify `BACK` if you want the model to face forward for retargeting/exporting glTF.

Reply:
I disagree, it's not tied to glTF. Here's a PR with just the look_at changes: https://github.com/godotengine/godot/pull/75875

Reply:
> Or you could have -Z forward in glTF if you are controlling a glTF camera which has its forward pointing -Z.

Well, even if the enum GLTF_FORWARD is an exaggeration, it is a definition that all of the assets as a GLTF are +Z, and a GLTF with -Z as the front is not a GLTF. A GLTF with a -Z front is not a GLTF. It is what is called a fake GLTF. GLTF cameras are out of scope here.

There may be a better enum (e.g. ASSET_FORWARD), but even if you can't find it, you must document somewhere that as long as Godot uses the GLTF importer for all formats and has a GLTF exporter, the asset must be +Z oriented. You have to document somewhere that the asset needs to be +Z oriented. (Root motion is also needed that document.)